### PR TITLE
now is able to get xml for county pages. still need to parse

### PR DESCRIPTION
**Pull request by lennybronner**
## Description
Previously `elex-clarity` didn't allow us to get precinct level results from a county specific clarity page like this one: https://results.enr.clarityelections.com/CA/Santa_Clara/110455/web.278093/#/summary. It threw an error since it assumed that there should be more than one county (if we wanted precinct results). 

We're now able to get precinct level results for such pages.

## Test Steps
`elexclarity elexclarity 110455 CA --countyName Santa_Clara --level=precinct`

### CODE REVIEW

File: src/elexclarity/formatters/const.py [hunk 1]
```diff
@@ -12,5 +12,7 @@
     "WV": {
         'PRESIDENT': 'P',
         'U.S. SENATOR': 'S'
+    },
+    "CA": {
```
Comment (emilyliu7321) [inline src/elexclarity/formatters/const.py:16]: If you wanted clarity IDs to be compatible with our existing race IDs like for GA and WV above, you could define this constant as  
```
"CA": {
        'Recall Gavin Newsom': "I_8808",
        'Office of the Governor Successor to Recall': 'G_recall_choice'
    }
```

File: tests/test_client.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
+from typing import NoReturn
```
Comment (emilyliu7321) [inline tests/test_client.py:1]: Unused import?

File: src/elexclarity/formatters/const.py (hunk 1 above)
Comment (lennybronner) [reply src/elexclarity/formatters/const.py:16]: Yeah, the issue is that the names are different for every county. In Contra Costa the race is called "Shall Gavin Newsom be recalled as Governor?" and in Marin it's just "Recall Question". So I decided I'll just do this on the post processing.

**Review by anthonyjpesce — CHANGES_REQUESTED** [review]
Two comments, the tox one is probably more important.

I got a little nervous that we were testing with 0s, so I checked the presidential election results `elexclarity 106043 CA --countyName Santa_Clara --level=precinct > test.json` and they looked ok (though you might want to double check).

I also went through the readme to try the examples from this branch and a few of them seem out of date or broken... but it doesn't look like this PR broke them 🙃 so I'm going to call it not your problem right now.

Do you mind updating the readme with `countyName` and an example from this PR though?

File: src/elexclarity/client.py [hunk 3]
```diff
@@ -54,14 +54,25 @@ def make_request(self, route, **kwargs):
...
+        url_subdirectory = f"{statepostal}/"
+        if countyname:
+            url_subdirectory += f"{countyname}/{electionid}"
+        else:
+            url_subdirectory += str(electionid)
+        return url_subdirectory
+
+    def get_current_version(self, electionid, statepostal, countyname):
+        url_subdirectory = self._get_url_subdirectories(electionid, statepostal, countyname)
+        current_ver_url = f"{url_subdirectory}/current_ver.txt"
```
[the remark this answers is not in the document]
Comment (lennybronner) [reply src/elexclarity/client.py:67]: Yeah, I wanted to add some testing for this generally, but adding the fixtures seemed like a lot fo work and I still need to write another 5 or so parsers for the counties that aren't using clarity. I... uhh... promise I'll add some after the election? Or if I finish the parsers earlier than expected this weekend.

**Review by emilyliu7321 — APPROVED** [review]

**Review by anthonyjpesce — APPROVED** [review]

**Pull request merged**